CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The following changes have been implemented but not released yet:
 
 ## [Unreleased]
 
+### New features
+
+- `access_api`: A set of APIs to manage Access which are agnostic to the Access
```

Comment:
Sorry, the ticket was unclear here, but since this object is a workaround for people who aren't able to use exports maps, as part of this PR we should also make the universal access module available through the exports map (i.e. using the "New feature" PR checklist).

And since that's the recommended way to import it (if your tooling supports it, which most tooling will as time goes by), we should just mention that module here and only mention this object in the docs with a note on compatibility.

And maybe explicitly mention that the Access Control mechanism currently supported are the only two mechanisms for Solid currently known, i.e. Web Access Control and Access Control Policies. Or would that be too detailed and something better suited for a note somewhere in the docs?

Reply:
Fixed respectively in de729cf17da372ad5b87e27a1b4be238ab95e6c5 and 35a8b79d33792e4e0b2565da636a76cff5aa0b29. Thanks for the detailed explanations !